tests/driver_dcf77/Makefile

```diff
@@ -0,0 +1,8 @@
+include ../Makefile.tests_common
+
+USEMODULE += dcf77
+USEMODULE += xtimer
+USEMODULE += periph_gpio_irq
```

Comment:
Just add an entry to `drivers/Makefile.dep`:

```Makefile
ifneq (,$(filter dcf77,$(USEMODULE)))
  FEATURES_REQUIRED += periph_gpio_irq
  USEMODULE += xtimer
endif
```

Then you only have to `USEMODULE += dcf77` and the dependencies will be added automatically.